OpenSCAP cloudapi tailoring

kingsleyzissou opened the pull request:
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

<!--
Thanks for proposing a change to osbuild-composer!

Please don't remove the above check list. These are things that each pull
request must have before it is merged. It helps maintainers to not forget
anything.

If the reason for ticking any of the boxes is ambiguous, please add a short
note explaining why.

In addition, if this pull request fixes a downstream issue, please refer to
test/README.md and add these additional items:

- [ ] 1st commit of any `rhbz#` related PR contains bug reproducer; CI reports FAIL or
- [ ] PR contains automated tests for new functionality and
- [ ] QE has approved reproducer/new tests and
- [ ] Subsequent commits provide bug fixes without modifying the reproducer; CI reports PASS and
- [ ] QE approves this PR; RHBZ status is set to `MODIFIED + Verified=Tested`

Information regarding our GitLab pipeline (Schutzbot):

CI will not be ran automatically if WIP label is applied or the PR is in DRAFT state, instead only
a link will be provided to the pipeline which can then be triggered manually if desired. To run the
CI automatically either switch the PR to ready or apply WIP+test label.

Outside contributors need manual approval from one of the osbuild-composer maintainers.

Schutzbot will only be triggered if all Tests jobs in GitHub workflow succeed.
-->

bcl commented on test/cases/oscap.sh:
```diff
@@ -480,7 +486,7 @@ fi
-if [[ ${SEVERITY} -gt 1 ]]; then
+if [[ ${SEVERITY} -gt 0 ]]; then
   redprint "❌ Failed"
   echo "More than one oscap rule with high severity failed"
```
Wording needs to be updated as well.

croissanne replied:
I guess this is still an issue?

kingsleyzissou replied:
Ah flip, I must have reverted the changes when I lost the commits

kingsleyzissou replied:
Ah no the diff shows it's been removed. But it's not showing on the review comment weirdly enough.

kingsleyzissou replied:
Ah flip haha I'm blind. I was too focussed on the diff

kingsleyzissou commented:
Not sure how I managed to close this. I will have to create a new PR

kingsleyzissou commented:
Somehow I managed to close this PR when I rebased it. :shrug: re-opening

kingsleyzissou commented:
Somehow I managed to close this PR when I rebased it. 🤷 re-opening

henrywang commented:
Test `edge-commit-cs9` failure is cause by bug https://bugzilla.redhat.com/show_bug.cgi?id=2234390. I have a workaround PR #3663.

croissanne reviewed (commented):
lgtm, @bcl's comment still stands though.

thozza reviewed (commented):
Except for my and other comments the PR looks good.

kingsleyzissou commented:
So I've addressed Brian's comment... but the diff in the comment hasn't updated (and won't update).
But if you look at the commit:

![image](https://github.com/osbuild/osbuild-composer/assets/20438192/c8f287a2-fc5f-4df0-80f5-b5e36e2eafce)

thozza reviewed (approved):
LGTM, nice work :slightly_smiling_face:

croissanne commented:
> So I've addressed Brian's comment... but the diff in the comment hasn't updated (and won't update). But if you look at the commit:
> 
> ![image](https://user-images.githubusercontent.com/20438192/264986211-c8f287a2-fc5f-4df0-80f5-b5e36e2eafce.png)

oh I thought it had to do with the comment "More than one oscap rule....", even though it's `-gt 0` now.

But yea not a big deal.

kingsleyzissou commented:
> > So I've addressed Brian's comment... but the diff in the comment hasn't updated (and won't update). But if you look at the commit:
> > ![image](https://user-images.githubusercontent.com/20438192/264986211-c8f287a2-fc5f-4df0-80f5-b5e36e2eafce.png)
> 
> oh I thought it had to do with the comment "More than one oscap rule....", even though it's `-gt 0` now.
> 
> But yea not a big deal.

Thanks, yeah no I completely missed the printed part. I fixed it though now :laughing:
